x/crud/internal/types/querier.go

```diff
@@ -21,3 +21,11 @@ type QueryResultRead struct {
 func (r QueryResultRead) String() string {
 	return r.Value
 }
+
+type QueryResultHas struct {
```

Comment:
Cool, we need a place to hold the query result. And a nice member function to get the value.